Elevation/Elevation.d.ts

```diff
@@ -0,0 +1,33 @@
+import MaterialComponent from '../MaterialComponent';
+import { VNode } from 'preact';
+
+export interface IElevationProps extends JSX.HTMLAttributes {
+  'z'?: number;
+  'z0'?: boolean;
```

Comment:
@prateekbh is it really helpful to have this props `z0` over having just z={0}?

Reply:
we can omit this in TS it's there just to avoid breaking change